android/src/main/java/com/gantix/JailMonkey/Rooted/GreaterThan23.java

```diff
@@ -31,7 +31,17 @@ private boolean checkRootMethod1() {
     private boolean checkRootMethod2() {
         Process process = null;
         try {
-            process = Runtime.getRuntime().exec(new String[] { "/system/xbin/which", "su" });
+            final String whichPathOld = "system/xbin/which";
+            final String whichPathNew = "system/bin/which";
+
+            if (new File(whichPathOld).exists()) {
+                process = Runtime.getRuntime().exec(new String[]{whichPathOld, "su"});
+            } else if (new File(whichPathNew).exists()) {
+                process = Runtime.getRuntime().exec(new String[]{whichPathNew, "su"});
+            } else {
+                process = Runtime.getRuntime().exec(new String[]{"which", "su"});
+            }
+
```

Comment:
Don't we need to do a similar kind of thing on `LessThan23.java` as well (https://github.com/GantMan/jail-monkey/blob/master/android/src/main/java/com/gantix/JailMonkey/Rooted/LessThan23.java)

Reply:
Plum doesn't support less than 23 I think.